packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -488,13 +488,12 @@ export class DocumentDeltaConnection
         // - a string: log it in the message (if not a string, it may contain PII but will print as [object Object])
         // - a socketError: add it to the OdspError object for driver to be able to parse it and reason
         //   over it.
-        let message = `socket.io: ${handler}`;
-        if (typeof error === "string") {
-            message = `${message}: ${error}`;
-        }
+        const message = `socket.io: ${handler}`;
         const errorObj = createGenericNetworkError(
             message,
             canRetry,
+            undefined,
+            { error: typeof error === "string" ? Object(error) : "" },
```

Comment:
I think we will lose info here. Will error be logged to telemetry? If not, we would definitely lose here, as errors from PUSH are very informative to understanding if it's a PUSH bug or client bug

Reply:
Before my change we were only adding it if it was a string as the comment says it can container PII.

Reply:
Can you please trace the code? If errors are coming from ordering service, then no, they should not contain any PII.
Even if in future it can be, we should leverage our telemetry annotation system to log it (as separate property) but let host decide to drop it or not. In internal rings we will not drop it, allowing developers deeper understanding of the problems.

Reply:
Reverted this specific change as mark already did it once and he didn't find it useful and then he reverted it.
https://github.com/microsoft/FluidFramework/pull/6675/files